product-service/swagger.yml

```diff
@@ -0,0 +1,88 @@
+swagger: "2.0"
+info:
+  description: "API documentation for AWS epam courses \n https://github.com/Lockd"
+  version: "1.0.0"
+  title: "Swagger for pottery shop"
+host: "z1tvhyxaxg.execute-api.eu-west-1.amazonaws.com"
+basePath: "/"
+tags:
+- name: "products"
+  description: ""
+schemes:
+- "https"
+- "http"
+paths:
+  /products:
+    get:
+      tags:
+      - "products"
+      summary: "Get list of all available products"
+      description: ""
+      operationId: "getProducts"
+      consumes:
+      - "application/json"
+      - "application/xml"
+      produces:
+      - "application/xml"
+      - "application/json"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/ListOfProducts"
+  
+  /products/id:
+    get:
+      tags:
+      - "products"
+      summary: "Get product by id"
+      operationId: 'getProductById'
+      consumes:
+      - "application/json"
+      - "application/xml"
+      produces:
+      - "application/xml"
+      - "application/json"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Product"
+        "404":
+          description: "item with selected id is not found"
+        
```

Comment:
Hey Lockd, the swagger is configured well 👍  , but maybe it's better to add the info about parameters like this 🤔 :

```yml
parameters: 
  - name: productId
    required: true
    in: path
    schema:
      type: string
```

Reply:
@yuleicul thanks, a good sugestion, added that!